tests/annotations/tests.py

```diff
@@ -1017,6 +1017,15 @@ def test_annotation_and_alias_filter_related_in_subquery(self):
             ],
         )
 
+    def test_annotation_exists_empty_query_result(self):
+        Company.objects.create(name="Django Software Foundation")
+        self.assertIs(
+            False,
+            Company.objects.annotate(exists=Exists(Company.objects.none()))
+            .get(name="Django Software Foundation")
```

Comment:
You can avoid the object creation entirely by using an existing `Author`

```suggestion
        self.assertIs(
            False,
            Author.objects.annotate(exists=Exists(Company.objects.none()))
            .get(pk=self.a1.pk)
```